policyengine_us/parameters/gov/states/va/tax/income/deductions/itemized/applicable_amount.yaml

```
@@ -1,4 +1,4 @@
-description: Virginia limits its itemized deductions to filers with federal adjusted gross income over this amount, based on filing status.
+description: Virginia limits the value of itemized deductions if federal AGI exceeds a given amount. The amount varies by filing status.
```

Avoid abbreviations in descriptions
```suggestion
description: Virginia limits the value of itemized deductions if federal adjusted gross income exceeds this amount, which varies by filing status.
```